bin/remove-undefined-collaborators.rb

```diff
@@ -29,6 +29,9 @@ def remove_collaborator(hash)
 url = ENV.fetch("OPS_ENG_REPORTS_URL")
 json = GithubCollaborators::HttpClient.new.fetch_json(url).body
 
+if json == "":
+  return
+
```

Comment:
don't think you need the : , but also, is this meant to be part of this PR?

Reply:
Duh - Python strikes again. Cheers for spotting.